src/components/ApplicantPreview/ApplicantPreview.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { UserResults } from '../../assets/definitions'
+
+const ApplicantPreview: React.FC<UserResults> = (props) => {
+  return (
+    <article className="applicant-cards">
```

Comment:
It would be cool to display a random bot from the DiceBear API based on the username on these ApplicantPreview cards. What do you think?

Reply:
I agree. Maybe we could pass it as a prop and only have to fetch them once - although, it could potentially create a lot of fetch calls... (I wonder if DiceBear has a bulk endpoint?)

Reply:
Oh, nice thought. If they did, we could make a fetch call (maybe upon App load?) for all existing usernames in the database and store the images that way. If they don't, we could make that fetch call upon load of SearchResults / Applicant, depending on the path the user is on.

Reply:
I agree, Erin and Greyson! We can add an image to each user object based on id by creating a clean data file that adds the fetched bot to each user object when we fetch all the users or we can grab a random/unassigned images that populate the found applicants each time a search is made.